src/test/java/com/squareup/javapoet/UtilTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2015 Square, Inc.
```

Comment:
Should be a mass update for all files, imho.

On Sat, Jan 16, 2016 at 8:04 PM, Jesse Wilson <notifications@github.com>
wrote:

> In src/test/java/com/squareup/javapoet/UtilTest.java
> <https://github.com/square/javapoet/pull/403#discussion_r49935762>:
>
> > @@ -0,0 +1,73 @@
> > +/*
> > + * Copyright (C) 2015 Square, Inc.
>
> 2016
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/square/javapoet/pull/403/files#r49935762>.
>